components/org.wso2.carbon.identity.application.authenticator.google/src/main/java/org/wso2/carbon/identity/application/authenticator/google/GoogleOAuth2Authenticator.java

```diff
@@ -89,8 +91,16 @@ protected String mapIdToken(AuthenticationContext context, HttpServletRequest re
             Map<String, String> authenticatorProperties = context.getAuthenticatorProperties();
             String clientID = authenticatorProperties.get(OIDCAuthenticatorConstants.CLIENT_ID);
 
-            validateCSRF(request, clientID);
-            boolean validJWT = Utils.validateGoogleJWT(request.getParameter(CREDENTIAL), clientID);
+            boolean internalSubmission = Boolean.parseBoolean(request.getParameter(INTERNAL_SUBMISSION));
+
+            LOG.info( "Validating the JWT submitted " + ( internalSubmission ? "internally" : "externally"));
```

Comment:
Do we need this info log?

Reply:
Yes, just for early statges. Otherwise we have to put additional effort on troubleshooting.

Reply:
If it is for troubleshooting, why can't we make it a debug log?

Reply:
Reasoned cab90221ea86d39fa6e6daaddeec6cc41cd2181e